README.md

```diff
@@ -38,18 +36,26 @@ in C++ library.
    The algorithms APIs on the other hand have deviated from their original API (but their logic is the same as Numenta's).
    If you are porting your code to this codebase, please review the [API Changelog](API_CHANGELOG.md).
 
-# Installation
+## Installation
 
-## Binary releases
+### Binary releases
+
+If you want to use `htm.core` from Python, the easiest method is to install from [PyPI](https://test.pypi.org/project/htm.core/)
+```
+python -m pip install -i https://test.pypi.org/simple/ htm.core
```

Comment:
Don't we have our wheels in pypi.org yet?
I don't think the general public can access test.pypi.org.

Reply:
no, not yet. 
And there's not too much pressure (atleast until we iron all the bugs in wheels regarding compatibility), yes, anyone can install using the command above. Actually, it's now successfully used in community/NAB repo.